tests/functional/Teradata/ToFinal/SqlBuilderTest.php

```diff
@@ -55,46 +53,23 @@ protected function getTestDBName(): string
         return $buildPrefix . self::TEST_DB;
     }
 
-    public function testGetDedupCommand(): void
+    protected function getBuilder(): SqlBuilder
     {
-        $this->createTestDb();
-        $stageDef = $this->createStagingTableWithData();
-
-        $deduplicationDef = new TeradataTableDefinition(
-            $this->getTestDBName(),
-            '__temp_tempTable',
-            true,
-            new ColumnCollection([
-                TeradataColumn::createGenericColumn('col1'),
-                TeradataColumn::createGenericColumn('col2'),
-            ]),
-            [
-                'pk1',
-                'pk2',
-            ]
-        );
-        $qb = new TeradataTableQueryBuilder();
-        $this->connection->executeStatement($qb->getCreateTableCommandFromDefinition($deduplicationDef));
-
-        $sql = $this->getBuilder()->getDedupCommand(
-            $stageDef,
-            $deduplicationDef,
-            $deduplicationDef->getPrimaryKeysNames()
-        );
-        $testDbName = TeradataQuote::quoteSingleIdentifier($this->getTestDBName());
-        self::assertEquals(
-        // phpcs:ignore
-            sprintf('INSERT INTO %s."__temp_tempTable" ("col1", "col2") SELECT a."col1",a."col2" FROM (SELECT "col1", "col2", ROW_NUMBER() OVER (PARTITION BY "pk1","pk2" ORDER BY "pk1","pk2") AS "_row_number_" FROM %s."stagingTable") AS a WHERE a."_row_number_" = 1', $testDbName, $testDbName),
-            $sql
-        );
-        $this->connection->executeStatement($sql);
-        $result = $this->connection->fetchAllAssociative(sprintf(
-            'SELECT * FROM %s.%s',
-            TeradataQuote::quoteSingleIdentifier($deduplicationDef->getSchemaName()),
-            TeradataQuote::quoteSingleIdentifier($deduplicationDef->getTableName())
-        ));
+        return new SqlBuilder();
+    }
 
-        self::assertCount(2, $result);
+    // TODO do we need it?
```

Comment:
nechal bych to tu, ostatni backendy to maji... ale maji k tomu navic i nejaky testy 🤷

Reply:
zrusil jsem to, protoze 1. v `testGetDropTableIfExistsCommand` se to uz dela 2. pouzivalo se to jen tam